flask_frozen/__init__.py

```diff
@@ -28,6 +27,8 @@
 from contextlib import contextmanager
 from collections import Mapping, namedtuple
 from posixpath import relpath as posix_relpath
+import datetime
```

Comment:
I know it wasn't sorted before, but could you move this import to the top since `datetime` comes before `os.path` alphabetically?